modules/common/log/grafana-alloy.nix

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  pkgs,
+  config,
+  lib,
+  hostName,
+  ...
+}: let
+  admin-ip-port = "192.168.101.10:3100";
```

Comment:
Please use the `admin-vm` hostname from `/etc/hosts` instead of duplicating the IP here. Port is fine.

Reply:
Done.